pkg/migrate/cstor/pool.go

```diff
@@ -88,23 +89,81 @@ func (c *CSPCMigrator) Migrate(name, namespace string) error {
 	if err != nil {
 		return errors.Wrap(err, "error building openebs clientset")
 	}
-	err = c.validateCSPCOperator()
```

Comment:
do we not want to validate the cspc operator existence ?

Reply:
Moved it preMigrate step